compliance/collection/main.go

```diff
@@ -256,14 +260,24 @@ func main() {
 	// Set the random seed based on the current time.
 	rand.Seed(time.Now().UnixNano())
 
+	var niClient scannerV1.NodeInventoryServiceClient
+
 	if features.RHCOSNodeScanning.Enabled() {
 		// Start the prometheus metrics server
 		metrics.NewDefaultHTTPServer().RunForever()
 		metrics.GatherThrottleMetricsForever(metrics.ComplianceSubsystem.String())
+
+		// Set up Compliance <-> NodeInventory connection
+		niConn, err := grpc.Dial("127.0.0.1:8444", grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
The port should be configurable over Helm with default `8444`, I think.

Reply:
> The port should be configurable over Helm with default 8444, I think.

Right now I don't think it can, no. Scanner usually opens two gRPC listeners, one [hardcoded](https://github.com/stackrox/scanner/blob/581c726c4295a330ff71c4a3ddd757e5e92c3286/api/grpc/grpc.go#L23) on port `8444` for internal connections, and another on a configurable port provided in the `config.yaml` file. 

Since both containers will share the same pod, they are able to access each other's ports even if they are not mapped in the container template (AFAIK). So defaulting to `8444` here isn't a problem. 

However, since it isn't too much of effort, I would already extract this into an external config (like an env variable). This way, if scanner gRPC code changes (e.g. due to clair unification) we don't need to change this code in the future.